src/common/transformations/src/transformations/utils/utils.cpp

```diff
@@ -340,6 +340,19 @@ bool can_eliminate_eltwise_node(const std::shared_ptr<Node>& eltwise,
     }
     return true;
 }
+
+float cast_eps_to_float(double eps_d) {
+    auto eps_f = static_cast<float>(eps_d);
+    if (eps_d > 0.) {  // zero is fine; negative values have no sense
```

Comment:
Added a comment in a function description, this solution is used and approved for mvn decomposition.
Additional assert can break backward compatibility, so I would like to avoid significant changes in this helper.